tests/test_common.py

```diff
@@ -174,16 +174,13 @@ def test_has_lte_version() -> None:
     assert not has_lte_version("1.7b")
 
 
-def test_which_no_bin_found() -> None:
-    assert which("top")
-    assert which("top", default_paths=[])
-    assert not which("top", default_paths=[], append_env_path=False)
-    assert not which("top", default_paths=["/"], append_env_path=False)
-
-
 def test_tmux_cmd_raises_on_not_found() -> None:
+    previous_path = os.environ["PATH"]
+    os.environ["PATH"] = ""
     with pytest.raises(TmuxCommandNotFound):
-        tmux_cmd("-V", tmux_search_paths=[], append_env_path=False)
+        tmux_cmd("-V")
+
+    os.environ["PATH"] = previous_path
```

Comment:
Line 177-183:

[pytest](https://docs.pytest.org/)'s [`MonkeyPatch.setenv()`](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.MonkeyPatch.setenv)

```python
def test_tmux_cmd_raises_on_not_found(monkeypatch: pytest.MonkeyPatch) -> None:
    monkeypatch.setenv("PATH", "")
    with pytest.raises(TmuxCommandNotFound):
        tmux_cmd("-V")
```

Pushed a change for this

Reply:
Sorry missed it :/
Let me apply it back